.github/CODEOWNERS

```diff
@@ -81,6 +81,7 @@ receiver/carbonreceiver/                             @open-telemetry/collector-c
 receiver/cloudfoundryreceiver/                       @open-telemetry/collector-contrib-approvers @agoallikmaa @pellared
 receiver/collectdreceiver/                           @open-telemetry/collector-contrib-approvers @owais
 receiver/dockerstatsreceiver/                        @open-telemetry/collector-contrib-approvers @rmfitzpatrick
+receiver/datadogreceiver                             @open-telemetry/collector-contrib-approvers @boostchicken
```

Comment:
Would you mind adding me to this too? I'd like to stay up to date with what happens.